prime-dotnet-webapi/Models/Enrollee.cs

```diff
@@ -67,6 +67,8 @@ public Enrollee()
 
         public ICollection<EnrolleeCareSetting> EnrolleeCareSettings { get; set; }
 
+        public ICollection<EnrolleeDeviceProvider> EnrolleeDeviceProviders { get; set; }
```

Comment:
Why is this a 1-to-many relationship?

Reply:
Can you add a comment with some specifics?

Reply:
added comment